docs/products/calloption.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Call Options
+sidebar_label: Call Options
+---
+A call option is a building block for the derivatives market. It gives the buyer the right but not the obligation to buy the asset at a specific price (strike) at a specific time. 
+
+This page will detail how to create your own call options and layout a few reasons why you would want to create call options.
+
+## How options work
+
+A trader buys a call option with a strike price of 2,000 USD for 0.05 ETH. Now he has the right to buy 1 ETH for 2,000 USD. At the expiry, ETH/USD is at 2,500 USD, and the delivery price is 2,500 USD.
+
+In this case, the option is settled for 500 USD per 1 ETH. At the expiry, the trader’s account is credited with 0.2 ETH (500/2,500), and the seller’s account is debited with 0.2 ETH. The initial purchase price was 0.05 ETH; therefore, the trader’s profit is 0.15 ETH.
+Any call option with an exercise price (strike price) above 2,500 USD will expire worthless.
+
+If the price of the option was less than $2000 the option will expire worthless. The payout function can be seen as:
+
+![Diagram](/img/option_payout.png)
+
+
+## Create your own options
+
+Using UMA's infrastructure a team can create a call option using our EMP contract template. These would allow a team to create European style options which are only redeemable at the time of expiry. 
+
+To create your own options you can make use of one of the currently approved [price identifiers](/uma-tokenholders/approved-price-identifiers.md) and [collateral types](uma-tokenholders/approved-collateral-currencies.md). 
```

Comment:
I think this should be expanded with more directions. It'd be great if this could be a step by step guide similar to what you wrote for the KPI options doc.

General structure of something like:

1. Are necessary price ids and collateral types approved? 
2. If no, go here for templates
3. Write (or use) a covered call financial product library. Code seen here: https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/common/financial-product-libraries/CoveredCallFinancialProductLibrary.sol
4. Launch an EMP with this tutorial. Use the covered call FPL from step 3.
5. Set your strike price by doing XYZ
6. Determine your option's starting price with Black-Scholes and create a market

Thoughts?